## 12.7 Hypothesis testing 

To test the null hypothesis $H_0: \beta_1=B$ against the alternative $H_1: \beta_1 \neq B$. We calculate the t-test statistic:

$$
t = \frac{(\hat{\beta_1}-B)}{SE(\hat{\beta_1})}
$$

Where $SE()$ denotes standard error. The standard error of an estimated regression coefficient is equal to the estimated standard deviation (i.e. we replace $\sigma$ with $\hat{\sigma}$ in the equation for $SD(\hat{\beta_1})$ above):

$$
SE(\hat{\beta_1})=\sqrt{\frac{\hat{\sigma}^2}{ns_x^2}}.
$$

Replacing $\sigma^2$ with $\hat{\sigma}^2$ means that $t$ follows a $t$ distribution (rather than a $z$ distribution) with $(n-2)$ degrees of freedom, if $H_0$ is true. This allows us to calculate the $p$-value to test the null hypothesis that $\beta_1$ equals $B$. 

Most commonly, researchers test the hypothesis that $\beta_1=0$. If this is true, then there is no association between $Y$ and $X$. 

**Example:** We can use the output from ```summary(model1)``` to conduct a hypothesis test to test the hypothesis that $\beta_1=0$ in Model 1:

In [1]:
#Example 1: Hypothesis tests
data<- read.csv('https://www.inferentialthinking.com/data/baby.csv')
model1<-lm(Birth.Weight~Gestational.Days, data=data)
summary(model1)


Call:
lm(formula = Birth.Weight ~ Gestational.Days, data = data)

Residuals:
    Min      1Q  Median      3Q     Max 
-49.348 -11.065   0.218  10.101  57.704 

Coefficients:
                  Estimate Std. Error t value Pr(>|t|)    
(Intercept)      -10.75414    8.53693   -1.26    0.208    
Gestational.Days   0.46656    0.03054   15.28   <2e-16 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 16.74 on 1172 degrees of freedom
Multiple R-squared:  0.1661,	Adjusted R-squared:  0.1654 
F-statistic: 233.4 on 1 and 1172 DF,  p-value: < 2.2e-16


In the above output, the column ```Std.Error``` gives the standard errors associated with the estimated regression coefficients. The columns ```t value``` and ```Pr(>|t|)``` give the t-test statistic and associated $p$-value for a hypothesis test that the regression coefficient estimate is equal to 0. 

To test the null hypothesis that $\beta_1=0$ against the alternative $\beta_1 \neq  0$, the test statistic is 15.28 and the associated $p$-value is $<2\times10^{-16}$. This is a very small $p$-value and therefore the data provide strong evidence against the null hypothesis. Based on these results, we can conclude that birthweight is associated with length of pregnancy. To convince yourself that these values are correct, you can calculate the standard error and test statistic by hand, using the above formulas. 

*Exercise:* Using the output for ```summary(model2)``` given in the previous section, conduct a hypothesis test to test the null hypothesis that $\alpha_1=0$.
 